src/blockencodings.h

```diff
@@ -22,7 +22,8 @@ struct TransactionCompressor {
 
     template <typename Stream, typename Operation>
     inline void SerializationOp(Stream& s, Operation ser_action, int nType, int nVersion) {
-        READWRITE(tx); //TODO: Compress tx encoding
+        // Future transaction compression standards could be implemented here
```

Comment:
It's not something someone looking at that code can do, it needs a proposal
first. So yes, it's a TODO but not at the code level.